presto-main/src/main/java/com/facebook/presto/server/BasicQueryInfo.java

```diff
@@ -46,12 +47,14 @@
     private final int queuedDrivers;
     private final int completedDrivers;
     private final int totalDrivers;
+    private final ErrorType errorType;
```

Comment:
I have incorporated this change but I have some additional though.
ErrorType is a derived value from ErrorCode using a stateless utility method. So having ErrorType and ErrorCode will be duplicity of data. But to transport any information to the UI, we need to have a method for the JSON conversion. Shall I just introduce the method to getErrorType and remove the field ErrorType.